permissions/plugin-icedq.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "icedq"
+github: "jenkinsci/icedq-plugin"
+paths:
+- "com/icedq/ci/plugin/icedq"
+developers:
+- "devlopertoranainc"
+- "toranainc"
+- "DevloperToranaInc"
```

Comment:
You're adding the same user twice with different cases, why?!

Reply:
i have account in jenkins jira with user name devlopertoranainc that's why im change my Yml file ,i don't have Account with username toranainc on jenkins jira ,suggest me what can i do for it ?